bdsx/assembler.ts

```diff
@@ -3261,6 +3270,16 @@ export class X64Assembler {
         }
         js.tab(4);
 
+        const addSourceMap = (id: Identifier, name: string, columnOffset: number): void => {
```

Comment:
I wish to remove `name`, unused parameter.

Reply:
Sorry, that unused parameter was leftover from my testing and I forgot to remove it.